internal/xds/v3/snapshotter.go

```diff
@@ -1,56 +0,0 @@
-// Copyright Project Contour Authors
```

Comment:
This seemed like an unnecessary layer of indirection between the snapshot handler and snapshot cache. Those two are now directly connected.